app/build.gradle

```diff
@@ -42,6 +42,8 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.0.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0'
+    implementation 'androidx.lifecycle:lifecycle-viewmodel-savedstate:2.2.0'
```

Comment:
It might be good to use the latest version of this, what do you think?

Reply:
I've updated the change, please check.

Reply:
Looks good!